WORKSPACE

```diff
@@ -38,6 +38,12 @@ load("//mixer:adapter_author_deps.bzl", "mixer_adapter_repositories")
 
 mixer_adapter_repositories()
 
+go_repository(
```

Comment:
btw, these things are sometimes auto-imported as part of rules_go.
https://github.com/istio/istio/blob/master/pilot/bin/check.sh#L10